esp/scm/ws_workunits.ecm

```diff
@@ -2237,15 +2237,15 @@ ESPservice [
     ESPmethod [cache_seconds(60)] WUMultiQuerysetDetails(WUMultiQuerySetDetailsRequest, WUMultiQuerySetDetailsResponse);
     ESPmethod [min_ver("1.71")] WUQuerysetImport(WUQuerysetImportRequest, WUQuerysetImportResponse);
```

Comment:
Should this invalidate the cache.  I think there should be a different cache group for the query sets from the workunits (and other infromation) since changes to one will not affect the other.

Reply:
Agreed, yes, different group and this would invalidate all.  This API can either add queries or completely replace the existing queryset.